src/main/java/pl/bykowsi/kurs/tydzien1pd/service/Basket.java

```diff
@@ -0,0 +1,53 @@
+package pl.bykowsi.kurs.tydzien1pd.service;
+
+import org.apache.commons.math3.random.RandomGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import pl.bykowsi.kurs.tydzien1pd.model.Product;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class Basket {
+
+
+    private static final Integer quantityOfProducts = 5;
+    private final RandomGenerator randomGenerator;
+    private final int MaxPrice;
+    private final int MinPrice;
+
+    @Autowired
+    private Basket(RandomGenerator randomGenerator, @Value("${price.MaxPrice}") int maxPrice, @Value("${price.MinPrice}") int minPrice) {
+        this.randomGenerator = randomGenerator;
+        this.MaxPrice = maxPrice;
+        this.MinPrice = minPrice;
+    }
+
+    public List<Product> createAndGetRandomBasket() {
+        List<Product> defaultList = createListOfDifferentProducts();
+        List<Product> finalList = new ArrayList<>();
+        for (int i = 0; i < quantityOfProducts; i++) {
```

Comment:
`for (int i = 0; i < quantityOfProducts; i++) {
            int productIndex = randomGenerator.nextInt(defaultList.size());
            finalList.add(defaultList.get(productIndex));
        }` można wydzielić do innej metody